tests/pytorch/test_pytorch_metric_value_convertion_utils.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from unittest import mock
+
+import mlflow
+from mlflow import tracking
+from mlflow.tracking.fluent import start_run
+from mlflow.exceptions import MlflowException, INVALID_PARAMETER_VALUE, ErrorCode
+from mlflow.tracking.metric_value_conversion_utils import *
+
+import torch
+
+
+def test_reraised_value_errors():
+    multi_item_torch_tensor = torch.rand((2, 2))
+
+    with pytest.raises(MlflowException) as e:
+        convert_metric_value_to_float_if_possible(multi_item_torch_tensor)
+
+    assert e.value.error_code == ErrorCode.Name(INVALID_PARAMETER_VALUE)
+
+
+def test_convert_metric_value_to_float():
+    torch_tensor_val = torch.rand(1)
+    assert float(torch_tensor_val[0]) == convert_metric_value_to_float_if_possible(torch_tensor_val)
```

Comment:
```suggestion
    assert convert_metric_value_to_float_if_possible(torch_tensor_val) == float(torch_tensor_val[0])
```

Can we move the actual value to the left?

Reply:
Can we also apply this change to other assert statements?